tests/kernel/timer/timer_monotonic/testcase.yaml

```diff
@@ -1,3 +1,9 @@
 tests:
   kernel.timer.monotonic:
     tags: timer
+  kernel.timer.stm32:
+    filter: dt_compat_enabled("stm32,lptim")
+    platform_whitelist: nucleo_wb55rg, nucleo_l4r5zi
```

Comment:
platform white listing is not needed following check on compatible

Reply:
removing the      `platform_whitelist: nucleo_wb55rg, nucleo_l4r5zi`